holistics/holistics.py

```diff
@@ -27,58 +27,51 @@ def __init__(self, api_key, path = None, url = None):
     def info(self):
         return {'api_key': self.headers['X-Holistics-Key'], 'path': self.path, 'url': self.url, }
 
-    def GetURL(self, tail_url, params=None):
-        try:
-            res = requests.get(self.url + tail_url, params = params, headers = self.headers)
-            res.raise_for_status()
-        except requests.exceptions.HTTPError as err:
-            print (err)
-            return 0
+    def get_url(self, tail_url, params=None):
+        res = requests.get(self.url + tail_url, params = params, headers = self.headers)
+        res.raise_for_status()
         return res
-        
 
-    def SubmitReport (self, report_id, filters=None):
+    def submit_report (self, report_id, filters=None):
         print ("Submitting export request... ", end='')
         tail_url = '/queries/'+str(report_id)+'/submit_export.csv'
-        res = self.GetURL(tail_url, filters)
-        if res!=0:
+        try:
+            res = self.get_url(tail_url, filters)
             res = res.json()
             print ("Success")
-            self.page['job_id'] = str(res['job_id'])
-            return 1
-        else:
-            return 0
+            return str(res['job_id'])
+        except requests.exceptions.HTTPError as err:
+            return err
```

Comment:
should not return error here. Either:
**Raise a new exception with friendlier message**:
```python
raise Exception('Fail to submit export')
```

To improve further, declare a new exception class like `HolisticsException` and do:
```python
raise HolisticsException('Fail to submit export')
```
The new exception class can be as specific as you want to describe the error appropriately

**Or do not catch the exception here, let `export_data` handle it**